soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -1265,9 +1294,6 @@ std::string FormatJsonHintText(std::string jsonHint) {
             formattedHintMessage.replace(start_pos, textToReplace.length(), iconString);
         }
     }
-
-    formattedHintMessage += 0x02;
-
     return formattedHintMessage;
```

Comment:
Just curious, why is this no longer needed?

Reply:
When accessing Path of Magic hints from using Saria's song, a certain control character (specifically `0x14`) is used to delegate control to the game so Navi can talk to you again and you can close the textbox when ready. Adding that character breaks that flow and causes a softlock.

Reply:
My worry is more about why this was here before. Does removing this not break anything else?

Reply:
From what I could tell, the game itself adds a `0x02` to signify EOF for messages. So most custom messages before had duplicate endings which made this safe to remove.